README.md

```diff
@@ -443,6 +443,11 @@ To ensure that a specific set of permissions are absent from the ACL, set `purge
 
  * When using SIDs for identities, autorequire tries to match to users with fully qualified names (e.g., User[BUILTIN\Administrators]) in addition to SIDs (User[S-1-5-32-544]). However, it can't match against 'User[Administrators]', because that could cause issues if domain accounts and local accounts share the same name e.g., 'Domain\Bob' and 'LOCAL\Bob'.
 
+ * When referring to accounts in the `APPLICATION PACKAGE AUTHORITY`, use either their SID values or their unqualified names. The Windows API has well documented bugs preventing the fully qualifed account names from being used.
```

Comment:
In Windows terms, I believe they're actually referred to as authorities, even though they look like domains.

Other authorities (including `SECURITY_APP_PACKAGE_AUTHORITY` (`{0,0,0,0,0,15}` AKA `S-1-15`)):
https://msdn.microsoft.com/en-us/library/dd302645.aspx

The new defs in WinNT.h (added for Windows 8) - https://recxltd.blogspot.com/2012/03/windows-8-app-container-security-notes.html